pkg/event/logger/handler.go

```diff
@@ -23,61 +27,72 @@ func (l *Handler) GetNetworkPlugin() string {
 
 func (l *Handler) TransitionToNonGateway() error {
 	klog.V(log.DEBUG).Info("The current node is no longer a Gateway")
+
 	return nil
 }
 
 func (l *Handler) TransitionToGateway() error {
 	klog.V(log.DEBUG).Info("The current node has become a Gateway")
+
 	return nil
 }
 
 func (l *Handler) LocalEndpointCreated(endpoint *submV1.Endpoint) error {
 	klog.V(log.DEBUG).Infof("A new Endpoint for the local cluster has been created: %#v", endpoint)
+
 	return nil
 }
 
 func (l *Handler) LocalEndpointUpdated(endpoint *submV1.Endpoint) error {
 	klog.V(log.DEBUG).Infof("The Endpoint for the local cluster has been updated: %#v", endpoint)
+
 	return nil
 }
 
 func (l *Handler) LocalEndpointRemoved(endpoint *submV1.Endpoint) error {
 	klog.V(log.DEBUG).Infof("The Endpoint for the local cluster has been removed: %#v", endpoint)
+
 	return nil
 }
 
 func (l *Handler) RemoteEndpointCreated(endpoint *submV1.Endpoint) error {
 	klog.V(log.DEBUG).Infof("A new Endpoint for remote cluster %q has been created: %#v",
 		endpoint.Spec.ClusterID, endpoint)
+
 	return nil
 }
 
 func (l *Handler) RemoteEndpointUpdated(endpoint *submV1.Endpoint) error {
 	klog.V(log.DEBUG).Infof("A new Endpoint for remote cluster %q has been updated: %#v",
 		endpoint.Spec.ClusterID, endpoint)
+
 	return nil
 }
 
 func (l *Handler) RemoteEndpointRemoved(endpoint *submV1.Endpoint) error {
 	klog.V(log.DEBUG).Infof("A new Endpoint for remote cluster %q has been removed: %#v",
 		endpoint.Spec.ClusterID, endpoint)
+
 	return nil
 }
 
-func (l *Handler) NodeAdded(node *k8sV1.Node) error {
+func (l *Handler) NodeCreated(node *k8sV1.Node) error {
 	klog.V(log.DEBUG).Infof("A Node with name %q and addresses %#v has been added to the cluster",
 		node.Name, node.Status.Addresses)
+
 	return nil
 }
 
 func (l *Handler) NodeUpdated(node *k8sV1.Node) error {
 	klog.V(log.DEBUG).Infof("A Node with name %q and addresses %#v has been updated",
 		node.Name, node.Status.Addresses)
+
 	return nil
 }
 
 func (l *Handler) NodeRemoved(node *k8sV1.Node) error {
 	klog.V(log.DEBUG).Infof("A Node with name %q has been removed",
 		node.Name)
+
```

Comment:
unrelated to this patch, my local linter was complaining I will remove those spaces.